ui/base-tool/src/main/kotlin/org/cru/godtools/base/tool/activity/BaseToolActivity.kt

```diff
@@ -171,8 +177,9 @@ abstract class BaseToolActivity<B : ViewDataBinding>(@LayoutRes contentLayoutId:
     protected open val shareMenuItemVisible by lazy { shareLinkUriLiveData.map { it != null } }
 
     protected open val shareLinkTitle get() = activeManifest?.title
+
     @get:StringRes
-    protected open val shareLinkMessageRes get() = R.string.share_tool_message
+    protected open val shareLinkMessageRes get() = R.string.share_tool_message_with_qr_code
```

Comment:
I think this is the only actual change in this file? everything else looks like auto-formatting changes